src/sc/asyncproofverifier.h

```diff
@@ -157,7 +157,9 @@ class TEST_FRIEND_CScAsyncProofVerifier
      */
     uint32_t GetMaxBatchVerifyDelay()
     {
-        return CScAsyncProofVerifier::GetInstance().BATCH_VERIFICATION_MAX_DELAY;
+        int32_t delay = GetArg("-scproofverificationdelay", CScAsyncProofVerifier::BATCH_VERIFICATION_MAX_DELAY);
+        assert(delay >= 0 && "scproofverificationdelay must be non negative");
+        return static_cast<uint32_t>(delay);
```

Comment:
In order to avoid code duplication, we could define a private getter method inside `CScAsyncProofVerifier` and use it both here and in `CScAsyncProofVerifier::RunPeriodicVerification()`.

Reply:
Done, I also turned assert() into condition check